packages/gluestick/src/commands/generate.js

```diff
@@ -1,6 +1,21 @@
+/* @flow */
+
+import type { Context } from '../types';
+
 const generator = require('../generator');
 
-module.exports = ({ config, logger }, generatorName, entityName, options) => {
+type GeneratorName = 'conatiner'
```

Comment:
Generator name can be any valid string, since there is possibility to create custom generator.